identity-service/src/routes/push_notifications.js

```diff
@@ -82,13 +100,21 @@ module.exports = function (app) {
       let params = { Token: deviceToken }
       if (deviceType === IOS) params = { ...iOSSNSParams, ...params }
       else if (deviceType === ANDROID) params = { ...androidSNSParams, ...params }
-      const awsARN = (await createPlatformEndpoint(params))['EndpointArn']
-      await models.NotificationDeviceToken.upsert({
-        deviceToken,
-        deviceType,
-        userId,
-        awsARN
-      })
+      if (deviceType !== SAFARI) {
```

Comment:
should this be else if?

Reply:
The previous if/else statements were for creating the aws sns params, but this if/else logic is for actually registering the device / saving to db.

Reply:
Will leave a comment in there though.